scripts/update_to_1.27.0.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+define('PACKAGE_URL', 'https://codeload.github.com/FreshRSS/FreshRSS/zip/1.27.0');
+define('MIN_PHP_VERSION', '8.1.0');
+
+
+// Apply the update by replacing old version of FreshRSS by the new one.
+function apply_update() {
+	$dirs_to_check = array(
+		DATA_PATH,
+		DATA_PATH . '/cache',
+		DATA_PATH . '/extensions-data',
+		DATA_PATH . '/favicons',
+		DATA_PATH . '/fever',
+		DATA_PATH . '/PubSubHubbub',
+		DATA_PATH . '/PubSubHubbub/feeds',
+		DATA_PATH . '/PubSubHubbub/keys',
+		DATA_PATH . '/Retry-After',
+		DATA_PATH . '/tokens',
+		DATA_PATH . '/users',
+		DATA_PATH . '/users/_',
+		FRESHRSS_PATH . '/extensions',
+	);
+
+	// check PHP version requirement
+	if (version_compare(MIN_PHP_VERSION, PHP_VERSION, '>')) {
+		return 'PHP Version ' . PHP_VERSION . ' not supported anymore. PHP required minimum ' . MIN_PHP_VERSION . '. Update aborted';
+	}
+
+	// First, do a backup.
+	$res = remove_data_backup();
+	if (!$res) {
+		return 'can\'t remove backup of ' . DATA_PATH;
+	}
+	$res = data_backup();
+	if (!$res) {
+		return 'can\'t do a backup of ' . DATA_PATH;
+	}
+
+	// For each directory, we check it exists, dir/index.html exists and we can
+	// write inside. (if the directory doesn't exist, it's created)
+	foreach ($dirs_to_check as $dir) {
+		$res = check_directory($dir);
+		if (!$res) {
+			return '`' . $dir . '` does not exist or FreshRSS cannot write inside';
+		}
+	}
+
+	// Get the FRSS package.
+	$res = save_package(PACKAGE_URL);
+	if (!$res) {
+		return 'can\'t save package ' . PACKAGE_URL;
+	}
+
+	// Deploy it on the server.
+	$res = deploy_package();
+	if (!$res) {
+		return 'can\'t deploy update package';
+	}
+
+	// And clean package files.
+	$res = clean_package();
+	if (!$res) {
+		return 'can\'t clean update package';
+	}
+
+	// bypass reauth prompt so that post_conf can trigger
+	Minz_Session::_param('lastReauth', time());
```

Comment:
I am not familiar with this variable, is it new in FreshRSS 1.27? And will it be required for the future updates?